universitas-negeri-semarang-fakultas-matematika-dan-ilmu-pengetahuan-alam.csl

```diff
@@ -149,8 +149,8 @@
               </else>
             </choose>
           </if>
-          <else-if type="thesis">
-            <text variable="genre" suffix="."/>
+          <else-if type="thesis"><!--Note digunakan untuk mensitasi skripsi, thesis dan disertasi-->
+            <text variable="note" suffix="."/>
```

Comment:
Yeah, "Type" still doesn't map to "genre" from Mendeley, oddly. But I do get a "Genre" field when creating a new Thesis item type. If you don't you can use Tools --> Options --> DOcument Details to add fields to item types in Mendeley.